src/a13-dates/verifiers/handler.js

```diff
@@ -0,0 +1,35 @@
+import { getCurrentFormattedDate } from './index.verify.js'
+import { mockCalls } from '../../../verify/test.verify.js'
+import { assertCalls } from '../../../verify/assert.verify.js'
+
+await import('../test/index.test.js')
+
+// I'm creating this wrapper in order to be able to spy the process.exit
+process.on('exit', () => {
```

Comment:
even though it isn't much code, can't we move all this, with the exception of what's specific to this step, to helper functions? to avoid repeating boilerplate code all the time. you can just have a funcion which accepts an array of "expectations" and takes care of the rest

Reply:
Good idea, will do it